vtr_flow/tasks/regression_tests/vtr_reg_strong/strong_scale_delay_budgets/config/config.txt

```diff
@@ -27,4 +24,4 @@ qor_parse_file=qor_fixed_chan_width.txt
 pass_requirements_file=pass_requirements_fixed_chan_width.txt
 
 # Script parameters
-script_params = --routing_budgets_algorithm scale_delay --route_chan_width 100
+script_params = --routing_budgets_algorithm scale_delay --route_chan_width 100 -sdc_file /sdc/samples/stereovision3.sdc
```

Comment:
Was it indented to add an SDC file here (since it wasn't specified before)?

Reply:
Yes, I tested the master branch and the sdc commands were applied.